manifest.konnector

```diff
@@ -28,7 +28,7 @@
     }
   },
   "data_types": [
-    "documents"
+    "document"
```

Comment:
This data type is not on data types listed this document: https://github.com/cozy/cozy-collect/blob/master/docs/konnector-manifest.md#data-types